src/components/PopupWithYes.js

```diff
@@ -0,0 +1,26 @@
+import Popup from './Popup.js';
+
+export default class PopupWithYes extends Popup{
```

Comment:
`PopupWithYes` should be called `PopupWithConfirmation` according to the check list